### DISCUSSION TITLE
SkuID & SkuPriceID in Focus Export

### DISCUSSION THREAD

**Discussion by DirkBrinkmann** [microsoft/finops-toolkit#729] [Q&A] [answered]
What are these SkuID and SkuPriceID in a Focus Export for an EA Customer?
It seems, they are not related to our Pricelist SkuIDs.

Thank you for any clarification.

![image](https://github.com/microsoft/finops-toolkit/assets/19854016/fa1dde51-b623-4f36-919e-040faa6de4e7)

**Comment by flanakin** [comment, marked as answer]
I just had a meeting about these today. Unfortunately, there's a gap in the dataset used to generate the cost and usage data in Cost Management, so we/they don't have the same fields as are on the price sheet for EA accounts. FOCUS is using internal IDs that cannot be joined with the price sheet today. I'm investigating what it'll take to get the same data and also proposing a FOCUS version of the price sheet. Both would be ideal.

If this is causing an issue for anyone, I would recommend raising this with a Microsoft account rep or raising a support request with Cost Management to help get this prioritized.
